src/tests/unit_asio_stream.cpp

```diff
@@ -177,7 +178,7 @@ class Asio_Stream_Tests final : public Test
          {
          net::io_context ioc;
          // fail right away
-         FailCount  fc{0, net::error::eof};
+         FailCount  fc{0, net::error::no_recovery};
```

Comment:
These were changed to avoid confusion when `eof` is mapped to `StreamTruncated`. The error code used here is chosen arbitrarily anyway.